pkg/meta/cassandra/handler/grpc/handler.go

```diff
@@ -48,10 +48,13 @@ func (s *server) GetMeta(ctx context.Context, key *payload.Meta_Key) (*payload.M
 	if err != nil {
 		if errors.IsErrCassandraNotFound(errors.UnWrapAll(err)) {
 			log.Warnf("[GetMeta]\tnot found\t%v\t%+v", key.GetKey(), err)
-			return nil, status.WrapWithNotFound(fmt.Sprintf("GetMeta API key %s not found", key.GetKey()), err, info.Get())
+			return nil, status.WrapWithNotFound(fmt.Sprintf("GetMeta API Cassandra key %s not found", key.GetKey()), err, info.Get())
+		} else if errors.IsErrCassandraUnavailable(errors.UnWrapAll(err)) {
+			log.Warnf("[GetMeta]\tunavailable\t%+v", err)
+			return nil, status.WrapWithUnavailable(fmt.Sprintf("GetMeta API Cassandra unavailable"), err, info.Get())
 		}
 		log.Errorf("[GetMeta]\tunknown error\t%+v", err)
-		return nil, status.WrapWithUnknown(fmt.Sprintf("GetMeta API unknown error occurred key %s", key.GetKey()), err, info.Get())
+		return nil, status.WrapWithUnknown(fmt.Sprintf("GetMeta API Cassandra unknown error occurred key %s", key.GetKey()), err, info.Get())
```

Comment:
`GetMeta API Cassandra` and `GetMeta API` were mixed in naming, so unify to `GetMeta API Cassandra` .